<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DjangoPage</class>
 <widget class="QWidget" name="DjangoPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>1201</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <widget class="QLabel" name="headerLabel">
     <property name="text">
      <string>&lt;b&gt;Configure Django&lt;/b&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line15">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="djangoConsoleGroup">
     <property name="title">
      <string>Console Commands</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Console Command:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="consoleCommandCombo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Enter the console command</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Console Command (non-closing):</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="consoleCommandNoCloseCombo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Enter the console command for a non-closing console</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>&lt;b&gt;Note:&lt;/b&gt; The console command for a console, that is spawning (i.e. exits before the console window is closed), should be prefixed by an '@' character.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="djangoServerGroup">
     <property name="title">
      <string>Server</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Server Address:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="serverAddressEdit">
        <property name="toolTip">
         <string>Enter the server port or server address and port (e.g. 127.0.0.1:8000 or [::1]:8000)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="ipv6CheckBox">
        <property name="toolTip">
         <string>Select to use IPv6</string>
        </property>
        <property name="text">
         <string>Use IPv6 protocol</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="djangoBrowserGroup">
     <property name="title">
      <string>Web-Browser</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QCheckBox" name="externalBrowserCheckBox">
        <property name="toolTip">
         <string>Select to use an external web-browser</string>
        </property>
        <property name="text">
         <string>Use external web-browser</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="djangoRecentAppsGroup">
     <property name="title">
      <string>Recent Applications Lists</string>
     </property>
     <layout class="QVBoxLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="_2">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Number of recent applications lists:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="appsRecentSpinBox">
          <property name="toolTip">
           <string>Enter the number of recent applications to remember</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="minimum">
           <number>5</number>
          </property>
          <property name="maximum">
           <number>50</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="python3Group">
     <property name="title">
      <string>Python 3</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QGroupBox" name="djangoVirtualEnvPy3Group">
        <property name="title">
         <string>Django Virtual Environment</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label_6">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>Enter the path of the Django virtual environment. Leave empty to not use a virtual environment setup.</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="virtualEnvPy3Edit"/>
         </item>
         <item row="1" column="1">
          <widget class="QToolButton" name="virtualEnvPy3Button">
           <property name="toolTip">
            <string>Select the virtual environment directory via a selection dialog</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="djangoPython3Group">
        <property name="title">
         <string>Django Python Console</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item>
          <widget class="QCheckBox" name="plainPython3CheckBox">
           <property name="toolTip">
            <string>Select to use plain Python instead of IPython</string>
           </property>
           <property name="text">
            <string>Use plain Python instead of IPython</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="python2Group">
     <property name="title">
      <string>Python 2</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGroupBox" name="djangoVirtualEnvPy2Group">
        <property name="title">
         <string>Django Virtual Environment</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label_7">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>Enter the path of the Django virtual environment. Leave empty to not use a virtual environment setup.</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="virtualEnvPy2Edit"/>
         </item>
         <item row="1" column="1">
          <widget class="QToolButton" name="virtualEnvPy2Button">
           <property name="toolTip">
            <string>Select the virtual environment directory via a selection dialog</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="djangoPython2Group">
        <property name="title">
         <string>Django Python Console</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item>
          <widget class="QCheckBox" name="plainPython2CheckBox">
           <property name="toolTip">
            <string>Select to use plain Python instead of IPython</string>
           </property>
           <property name="text">
            <string>Use plain Python instead of IPython</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="TranslationsGroup">
     <property name="title">
      <string>Translations Editor</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_8">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Enter the path of an editor to use to do the translations. Leave empty to disable this feature.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="translationsEdit"/>
      </item>
      <item row="1" column="1">
       <widget class="QToolButton" name="translationsButton">
        <property name="toolTip">
         <string>Select the translations editor via a file selection dialog</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>274</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>consoleCommandCombo</tabstop>
  <tabstop>consoleCommandNoCloseCombo</tabstop>
  <tabstop>serverAddressEdit</tabstop>
  <tabstop>ipv6CheckBox</tabstop>
  <tabstop>externalBrowserCheckBox</tabstop>
  <tabstop>appsRecentSpinBox</tabstop>
  <tabstop>virtualEnvPy3Edit</tabstop>
  <tabstop>virtualEnvPy3Button</tabstop>
  <tabstop>plainPython3CheckBox</tabstop>
  <tabstop>virtualEnvPy2Edit</tabstop>
  <tabstop>virtualEnvPy2Button</tabstop>
  <tabstop>plainPython2CheckBox</tabstop>
  <tabstop>translationsEdit</tabstop>
  <tabstop>translationsButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
